docs/platform/manager/accounts-and-passwords-new-manager/index.md

```diff
@@ -0,0 +1,162 @@
+
+    {{< note >}}
+Granting access to settings denoted with a dollar sign ($) will allow the user to perform actions that incur billing costs, such as adding or resizing a Linode.
+{{< /note >}}
+
+1.  Select the appropriate permissions (None, Read Only, or Read-Write) in the **Permissions** section to allow the user access your Linodes, StackSripts, Images, NodeBalancers and Domains.
+2.  When you have finished configuring the user's permissions, click **Save**. The user's permissions will be saved and effective immediately.
+
+### Recovering a Lost Username
+
+Did you forget your Linode Manager username? Recover it with the *Forgot Username* webpage. Here's how:
+
+1.  Visit the [Forgot Username](https://manager.linode.com/session/forgot/username) webpage.
+1.  Enter your email address in the **Email** field.
+1.  Click **Submit**.
+
+In a couple minutes, you'll receive an email message with any Linode Manager users that correspond to that email. If you do not receive information about any users, then you may have registered your account with a different email.
+
+### Removing a User
+
+You can permanently remove a user account from the Linode Manager. Here's how:
+
+1.  Click the **Account** link on the left-hand side of the Manager.
+2.  Click the **Users** tab.
+3.  Locate the user in the list and select the **Delete** link from the menu. A warning appears asking you to confirm that you want to delete the user.
+    ![Use the Linode Manager to delete a User](accounts-delete-a-user.png "Use the Linode Manager to delete a User.")
+4.  Click **Delete**.
+
+The account will be removed, and the user will no longer be able to access the Linode Manager.
+
+## Email Addresses and Contact Information
+
+Linode uses the contact information on file in your account to notify and bill you. Keep this information current to prevent service interruptions. It's especially important to keep your email address current.
+
+Both the *Account & Billing* and *My Profile* pages have an email address field. The email addresses saved on these pages receive different notifications, as described in the following sections. If you are the only user, you should enter your email address on both webpages. If there are multiple users, verify that the primary account holder's email address is current on the *Account* webpage.
+
+### Updating Contact Information
+
+Use the *Account & Billing* webpage to update the contact information for the Linode account. The email address saved on this webpage receives invoices, receipts, and credit card expiration warnings. Support tickets are *not* sent to this email address.
+
+Here's how to update the contact information and the email address on the *Account* webpage:
+
+1.  Click the **Account** link on the left-hand side of the Manager.
+1.  Click the **Account & Billing** tab.
+1.  Expand **Update Contact Information** and update the contact information and the email address for the account.
```

Comment:
This screenshot still has the old header. Not a huge deal, but if you wanted you could crop the top and all would be well.